src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OdsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Organisations;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
+{
+    //[Authorize(Policy = Policy.CanAccessOrganisation)]
```

Comment:
I was expecting this `[Authorize(Policy = Policy.CanAccessOrganisation)]` attribute to be included. This will ensure a user with the right privileges can access this data.

Reply:
My reasoning behind leaving it without auth was to make it easier for the FE folk to access this while the data is still canned.

If you feel like it should be 'protected' to start with, I'll un-comment line 5 :]